pyiceberg/catalog/glue.py

```diff
@@ -266,6 +289,10 @@ def __init__(self, name: str, **properties: Any):
         )
         self.glue: GlueClient = session.client("glue")
 
+        self.glue_catalog_id: Optional[str] = properties.get("glue.id")
+        if self.glue_catalog_id is not None:
+            _register_glue_catalog_id_with_glue_client(self.glue, self.glue_catalog_id)
```

Comment:
```suggestion
        if glue_catalog_id := properties.get(GLUE_ID):
            _register_glue_catalog_id_with_glue_client(self.glue, glue_catalog_id)
```
Since a valid glue ID cannot be an empty string, I think we could use walrus operator here. Also, seems we do not need to store glue_catalog_id in the class. The information can be retrieved from `self.properties`. WDYT?

Reply:
I'm fine with both of those.